.gitattributes

```diff
@@ -0,0 +1,16 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+*.c text
```

Comment:
This line, as well, as the one below (the ".h" pattern) are not needed, cause Nodezator doesn't have .c or .h files in its source. Of course, if someones later adds them, then there's no problem with setting such patterns here, but for now they aren't really needed.

It is best that we act upon our needs rather than when we only foresee that we will need something.